packages/terra-docs/src/Notice.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './Notice.module.scss';
+
+const cx = classNames.bind(styles);
+
+const getTitle = (variant) => {
+  if (variant === 'deprecation') {
+    return 'Deprecation Notice';
```

Comment:
I did not translate the notice title. None of our docs are translated so it would look weird if this switched an nothing else?

Reply:
Seems appropriate to me.

Reply:
Should we add that as a blurb in the comment ?